docs_versioned_docs/version-ros2jazzy/components/yaml/attachments/a300/wireless_charger.mdx

```diff
@@ -0,0 +1,29 @@
+<table>
+<tr>
+<td>
+<center>
+<figure>
+    <img src={require("./img/wireless_charger.png").default} width="250" />
+</figure>
+</center>
+</td>
+<td>
+
+```yaml
+platform:
+  attachments:
+    - name: wireless_charger
+      type: a300.wireless_charger
+      model: default
+      parent: base_link
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+      enabled: true
+```
+</td>
+</tr>
+</table>
+
+The A300 can optionally be equipped with a wireless charging coil, mounted to the the right side of the robot.
+
+The `${name}_face` link is located in the centre of the charging coil's face, with the X axis pointing outwards.
```

Comment:
- What's the purpose of the link on the end of the wireless charger's face? 
- Why rotate the coordinate system relative to base_link?